flow/project.py

```diff
@@ -761,32 +802,34 @@ def np_callable(job):
         directives.setdefault('omp_num_threads', 0)
         directives.setdefault('processor_fraction', 1)
 
-        # Evaluate strings and callables for job:
+        # Evaluate strings and callables for jobs:
         def evaluate(value):
             if value and callable(value):
-                return value(job)
+                return value(*jobs)
             elif isinstance(value, str):
-                return value.format(job=job)
+                return value.format(*jobs)
             else:
                 return value
         return {key: evaluate(value) for key, value in directives.items()}
 
-    def _submit_cmd(self, entrypoint, ignore_conditions, job=None):
-        entrypoint = self._determine_entrypoint(entrypoint, dict(), job)
+    def _submit_cmd(self, entrypoint, ignore_conditions, jobs=None):
+        entrypoint = self._determine_entrypoint(entrypoint, dict(), jobs)
         cmd = "{} run -o {}".format(entrypoint, self.name)
-        cmd = cmd if job is None else cmd + ' -j {}'.format(job)
+        cmd = cmd if jobs is None else cmd + ' -j {}'.format(' '.join(map(str, jobs)))
```

Comment:
This will work, but we may want to think on this before the finish aggregation, as this could lead to a huge command line command depending on number of jobs.

Reply:
Noted. Thanks for the suggestion.

Reply:
We'll be storing the submission id of the aggregate in the `.aggregates/` directory which is not introduced yet but will be introduced in [#MyFork/2](github.com/kidrahahjo/signac-flow/pull/2).
I think that will be enough for a meta-data?

Reply:
@kidrahahjo Bradley is referring to meta data is more than just the list of job ids. @bdice I do agree this is not a problem in the script working (long lines are not going to freeze up the scheduler). However, it would get inconvenient for users to look at their scripts. I am okay with waiting on this (I planned on doing that for this PR anyways), but I think raising an issue is important so we don't lose track of this.